nss-day-cohort-48/rare-server-river-rats
# Nwm login register

#### Changes Made
1. Added file login/__init__.py   login/request.py to directory.
1. Modified file rare_users/request.py and request_handler.py and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout branchname
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    updated the user to add get user and get all users and see them listed in postman
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
